agent/tests/test_api_internal.c

```diff
@@ -56,13 +56,14 @@ static void test_invalid_parameters(TSRMLS_D) {
 
   /* Literally any parameter should cause this to bail. */
 #ifdef PHP8
```

Comment:
Maybe a good opportunity to replace a "PHP8" macro with the preferred API number macro(s)?